docs/docset.yml

```
@@ -45,5 +46,5 @@ subs:
   stack-version: "9.0.0"
   major-version: "9.x"
   docker-repo: "docker.elastic.co/logstash/logstash"
-  ess-leadin-short:   "Our hosted Elasticsearch Service is available on AWS, GCP, and Azure, and you can try it for free at https://cloud.elastic.co/registration"
-  ess-leadin: "You can run Elasticsearch on your own hardware or use our hosted Elasticsearch Service that is available on AWS, GCP, and Azure. Try the Elasticsearch Service for free: https://cloud.elastic.co/registration."
+  ess-leadin-short:   "Elastic Cloud Hosted is available on AWS, GCP, and Azure, and you can try it for free at https://cloud.elastic.co/registration"
```

Maybe add a period here (as in the _ess-leadin_ substitution):

```suggestion
  ess-leadin-short:   "Elastic Cloud Hosted is available on AWS, GCP, and Azure, and you can try it for free at https://cloud.elastic.co/registration."
```

Also, can we make the URL in the substitution render as a link after the build?

---

Yes! Good catch. I added the period and then lost it on a rebase gone sideway.  :-)